SWLOR.Game.Server/Feature/AbilityDefinition/Force/DisturbanceAbilityDefinition.cs

```diff
@@ -25,15 +26,51 @@ public class DisturbanceAbilityDefinition : IAbilityListDefinition
             return _builder.Build();
         }
 
-        private void Impact(uint activator, uint target, int dmg, int accDecrease, int tier, string effectTag, int dc)
+        private static void ImpactAction(uint activator, uint target, int level, Location targetLocation)
         {
+            var willBonus = GetAbilityScore(activator, AbilityType.Willpower);
+            var dmg = 0;
+            string effectTag = "";
```

Comment:
I'm confused about the change to the signature here. The effect tag was passed in to avoid the need to do a case statement. You're also using the wrong effect tag below. Please restructure this back to how it was before for consistency purposes and also use the tag constants that are already defined in this file.